style.css

```diff
@@ -0,0 +1,467 @@
+  font-size: 30px;
+  font-weight: 500;
+  color: #edecff;
+  text-align: center;
+  padding: 30px 0 0 40px;
+  margin: 0;
+}
+
+h1 {
+  font-size: 38px;
+  font-weight: bold;
+  color: #7d73fc;
+  text-align: center;
+  padding: 60px 0 0 50px;
+  margin-bottom: 0;
+}
+
+.s {
+  padding-bottom: 20px;
+}
+
+.img {
+  padding: 0 30px;
+  align-items: center;
+}
+
+p {
+  font-size: 20px;
+  text-align: left;
+  vertical-align: top;
+  padding: 0 10px 0 30px;
+}
+
+.content p {
+  font-size: 20px;
+  text-align: center;
+  margin: 0;
+  padding: 50px 20px;
+}
+
+img:hover {
+  background: cyan;
+}
+
+h3 {
+  font-size: 32px;
+  text-align: left;
+  vertical-align: center;
+  padding: 30px;
+}
+
+.logo h3 {
+  color: #fff;
+  margin-top: 10px;
+  font-size: 20px;
+  text-decoration: none;
+}
+
+.media {
+  margin: 0 10px 0 0;
+}
+
+.arrow h3 {
+  color: #a2a3ae;
+  margin-left: 15px;
+  font-size: 20px;
+}
+
+.arrow {
+  display: flex;
+  text-align: center;
+  justify-content: center;
+}
+
+.placeholder {
+  display: flex;
+  flex-wrap: wrap;
+  text-align: center;
+  padding: 30px 0;
+  background: url("image/Ellipse19\(2\).svg");
+  background-repeat: no-repeat;
+  background-position: left;
+}
+
+.work {
+  color: #9490ff;
+  font-size: 40px;
+  text-align: left;
+  vertical-align: center;
+}
+
+.items {
+  display: flex;
+  justify-content: center;
+  padding-left: 20px;
+}
+
+.items li {
+  border-right: 1px solid #fff;
+  border-left: none;
+  border-top: none;
+  border-bottom: none;
+  font-size: 15px;
+  padding: 0 20px 0;
+  width: 50%;
+}
+
+.items .ruby {
+  border-right: none;
+}
+
+.button {
+  width: 89px;
+  height: 48px;
+  margin: 30px 30px 0;
+  background: #7d73fc;
+  color: #fff;
+  border: 0;
+  float: left;
+  font-size: 12px;
+  cursor: pointer;
+}
+
+.c {
+  color: #7d73fc;
+  text-align: center;
+}
+
+.background-image1 {
+  display: flex;
+  flex-wrap: wrap;
+  text-align: center;
+  padding: 30px 0;
+  background: url("image/Ellipse20.svg");
+  background-repeat: no-repeat;
+  background-position: right;
+}
+
+.work-section {
+  display: grid;
+}
+
+.background-image2 {
+  display: flex;
+  flex-wrap: wrap;
+  text-align: center;
+  padding: 30px 0;
+  background: url("image/Ellipse21.svg");
+  background-repeat: no-repeat;
+  background-position: left;
+}
+
+.background-image3 {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  padding: 30px 0;
+  background: url("image/Ellipse22.svg");
+  background-repeat: no-repeat;
+  background-position: right;
+}
+
+.button:hover {
+  background: #7d73fc;
+}
+
+.button:active {
+  background: #5e42d7;
+}
+
+.button:disabled {
+  background: #2a2b33;
+  cursor: not-allowed;
+}
+
+@media only screen and (min-width: 768px) {
+  .content {
+    vertical-align: top;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    background-image: url("images/circle1.svg");
+    background-repeat: no-repeat;
+    background-position: left;
+  }
+
```

Comment:
- [ ] Kindly fix the responsive nature of your app to fit different screen sizes as expected.
### **See Mobile View**
![mob](https://user-images.githubusercontent.com/55596970/167165990-95516d92-86ec-49cb-84f7-5ead44974a80.png)